src/styles/scss/home/contact.scss

```diff
@@ -38,15 +38,15 @@
   }
 }
 
-.contact-field-container[role=school] {
+.contact-field-container-school {
```

Comment:
For contact field containers, let's do the following:
- Change `contact-field-container-school` and `contact-field-container-email` back to one class: `contact-field-container`. 
- To avoid using attributes for styling, let's use this for the styling:
```css
.contact-field-container {
  
  &:first-child {
    padding-right: 9px;

    * {
      float: right;
    }
  }
  
  &:last-child {
    padding-left: 9px;
  
    * {
      float: left;
    }
  }
}
```
- Lastly, let's change `.contact-field-container[role]` to just `.contact-field-container` on line 146 of `contact.scss`.